### PR TITLE
Issue 30 : message headers failure

### PR DESCRIPTION
**Pull request by thibaultleouay**
#30

### CODE REVIEW

File: src/main/java/io/vertx/rabbitmq/impl/Utils.java [hunk 1]
```diff
@@ -65,6 +71,26 @@ public static JsonObject toJson(AMQP.BasicProperties properties) {
     return json;
   }
```
Comment (poiuytrez) [inline src/main/java/io/vertx/rabbitmq/impl/Utils.java]: Could you add the javadoc ?

Comment (waltersve) [reply src/main/java/io/vertx/rabbitmq/impl/Utils.java]: Hi what is the current of the issue? I'm facing the same issue and looking forward to the new version including the patch. If only the javadoc is preventing the merge I could provide something ;).

Comment (vietj) [reply src/main/java/io/vertx/rabbitmq/impl/Utils.java]: I believe there should be a test as well

Comment (vietj) [reply src/main/java/io/vertx/rabbitmq/impl/Utils.java]: but well @waltersve any contribution is welcome :-)

Comment (waltersve) [reply src/main/java/io/vertx/rabbitmq/impl/Utils.java]: Hi @vietj, I created a pull-request including the patch, javadoc and tests. I hope that work for you.

**Pull request closed without merging**